web/src/components/SearchFeedback.tsx

```diff
@@ -26,21 +44,35 @@ const SearchFeedback = ({ cityCode, languageCode, query, noResults }: SearchFeed
 
   useEffect(() => setShowFeedback(false), [query])
 
-  if (noResults || showFeedback) {
+  if (showFeedback) {
     return (
       <Container>
         <FeedbackContainer
           cityCode={cityCode}
           language={languageCode}
           routeType={SEARCH_ROUTE}
           query={query}
-          noResults={noResults}
           initialRating={null}
         />
       </Container>
     )
   }
 
+  if (noResults) {
+    const fallbackLanguage = config.sourceLanguage
+
+    return (
+      <CenteredContainer>
+        <SemiBoldText>
+          {languageCode === fallbackLanguage ? t('noResultsInOneLanguage') : t('noResultsInTwoLanguages')}
+        </SemiBoldText>
+        <MiddleText>{t('checkQuery', { appName: buildConfig().appName })}</MiddleText>
```

Comment:
:upside_down_face: I'd try not to give names based on positioning as that might change in the future and instead rather use names more related to function.
```suggestion
        <Hint>{t('checkQuery', { appName: buildConfig().appName })}</Hint>
```